frontend/src/container/TopNav/DateTimeSelection/index.tsx

```diff
@@ -295,6 +295,7 @@ function DateTimeSelection({
 						style={{
 							minWidth: 120,
 						}}
+						listHeight={400}
```

Comment:
How was the height working before this? Why do we need the explicit height ?

Reply:
<img width="1177" alt="Screenshot 2024-01-05 at 11 40 39 AM" src="https://github.com/SigNoz/signoz/assets/26198062/8a846a0b-6ffd-47ea-84dc-65f4ec41293c">

It was using the default height before.